python/ray/_private/services.py

```diff
@@ -1118,7 +1118,16 @@ def start_api_server(
                 else:
                     raise e
         # Make sure the process can start.
-        minimal = not ray._private.utils.check_dashboard_dependencies_installed()
+        minimal: bool = not ray._private.utils.check_dashboard_dependencies_installed()
+
+        # Explicitly check here that when the user explicitly specifies
+        # dashboard inclusion, the install is not minimal.
+        if include_dashboard and minimal:
+            logger.error("--include-dashboard was specified, but packages are missing.")
```

Comment:
nit: we could probably tell users to check `dashboard.err` for the specific ones that are missing.

Reply:
done